tests/fsharpqa/Source/Conformance/SpecialAttributesAndTypes/Imported/CallerInfo/CallerMemberName.fs

```diff
@@ -0,0 +1,150 @@
+namespace Test
+
+open System.Runtime.CompilerServices
+open System.Reflection
+open CSharpLib
+
+[<assembly:MyCallerMemberNameAttribute>]
+do
+    ()
+
+[<MyCallerMemberName()>]
+type MyTy() =
+    let functionVal = MyTy.GetCallerMemberName
+    let typeLetValue = MyTy.GetCallerMemberName()
+    let typeLetFunc (i:int) = i, MyTy.GetCallerMemberName()
+    let typeLetFuncNested () =
```

Comment:
@dsyme  I added the tests you asked I hope I understand you correctly
I think maybe the anonymous lambda expression result is wrong hwat do you think is the expected value?